.sass-lint.yml

```diff
@@ -114,7 +114,7 @@ rules:
   no-qualifying-elements:
     - 1
     - allow-element-with-attribute: false
-      allow-element-with-class: false
+      allow-element-with-class: true
```

Comment:
This is a bad practice and should stay at `false`. In cases like this one (when we know this is bad but, meh, there is no other solution), we should use `sass-lint:disable` instead.

Reply:
> we should use sass-lint:disable

You were against setting this per line in files and meant that we should only set the config options in the yml file. I read some issues regarding this rule and many recommend and use this setting.

Reply:
I was against using `sass-lint:disable` for `force-element-nesting` and `force-pseudo-nesting` because there can be many case when we want to avoid nesting (like: because we just have a single property to set in a deeply-nested element and don't want to take 10 lines for that). So instead of adding "exceptions" everything, let's change the rule because the rule is bad.

For `allow-element-with-class` I think it is very useful to force us to take care of the specificity. It does not depends on the way we want to organize our properties to get the best readibility but on the selector itself, the CSS that is exposed to the user and the whole architecture we (badly) try to follow: like avoiding heavy selectors, magical properties applied from the parent and everything leading to the specificity hell.

Does it makes sense to you ?